From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Anton Hansson <hansson@google.com>
Date: Thu, 8 Aug 2024 19:30:31 +0200
Subject: [PATCH 16/18] sm6225-common: Use generic arch variant

Follow blueline which has not demonstrated any regressions
after switching to a generic arch variant.

Bug: 120773446
Test: make
Change-Id: I9e23963fcfa5ccaa912f2930a632458c4d2a692d
---
 BoardConfigCommon.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 5f08ff0..5cf1653 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -16,7 +16,7 @@ TARGET_2ND_ARCH := arm
 TARGET_2ND_ARCH_VARIANT := armv8-a
 TARGET_2ND_CPU_ABI := armeabi-v7a
 TARGET_2ND_CPU_ABI2 := armeabi
-TARGET_2ND_CPU_VARIANT := cortex-a9
+TARGET_2ND_CPU_VARIANT := generic
 
 # Audio
 AUDIO_FEATURE_ENABLED_EXT_AMPLIFIER := true
